app/views/pages/notebooks.html.erb

```diff
@@ -0,0 +1,79 @@
+    <p class="about_p">After running <code>jupyter notebook</code>, the following window should appear:</p>
+    <%= image_tag("Notebook-03.png", alt: "The Jupyter Notebook opening page.", class:"body_img")%>
+    <p class="top_toc"><%=link_to('Back to Top', anchor: '#toc-top') %></p>
+
+    <h3 class="about_h3" id="3toc">Using the Notebooks</h3>
+    <p class="about_p">Congratulations!  You’ve now opened the Cloud Notebooks directory in Jupyter. There are three notebooks to choose from:</p>
+    <ul class="about_ul">
+      <li class="about_li"><strong>auk-notebook-domains</strong>: This notebook allows you to perform domain-level analysis on an archived collection. This includes analyzing which domains appear most frequently within the collection.</li>
+      <li class="about_li"><strong>auk-notebook-network</strong>: This notebook provides basic network analysis and visualization of linkages between domains within the collection.</li>
+      <li class="about_li"><strong>auk-notebook-text</strong>: This notebook uses Python’s Natural Language Toolkit (NLTK) to perform text analysis, including concordance analysis, sentiment analysis, and dispersion plots.</li>
+    </ul>
+    <p class="about_p">For more information on what each notebook can do, see Ryan Deschamps’s <%= link_to('"Exploring Web Archival Data through Archives Unleashed Cloud Jupyter Notebook."', 'https://news.archivesunleashed.org/exploring-web-archival-data-through-archives-unleashed-cloud-jupyter-notebooks-7605c6ca2b33', target: '_blank') %></p>
+    <p class="about_p">Notebooks contain code and visualizations like the following one from <strong>auk-notebook-network</strong>:</p>
+    <%= image_tag("Notebook-04.png", alt: "The Jupyter Notebook Network example.", class:"body_img")%>
+    <p class="about_p">In order to use each notebook, click on the notebook from the directory list — the notebook  will open in a new tab or window. Each notebook has been pre-loaded with sample data from the <%= link_to('B.C. Teachers\' Labor Dispute','https://archive-it.org/collections/4867', target: '_blank') %> (2014) collection from the University of Victoria Libraries. This  data is great to play with in order to learn how the tools work!  Once you are ready to load your own data, there are a few extra steps to load it into the notebooks.</p>
+    <p class="top_toc"><%=link_to('Back to Top', anchor: '#toc-top') %></p>
+
+    <h3 class="about_h3" id="4toc">Generating the Derivatives</h3>
+    <p class="about_p">In order to use the Cloud Notebooks with your own data, you will first need to analyze your collection using the Archives Unleashed Cloud, as described in the Cloud <%= link_to('documentation', '/documentation', target: '_blank') %>. This process can take several hours, depending on the queue of jobs running at the time. You will receive an email notification once the analysis is complete.</p>
```

Comment:
suggestion: "depending on the queue of jobs running at the time" --> drop "at the time"